app/controllers/standup_meetings_controller.rb

```diff
@@ -7,6 +7,10 @@ def index
     @standup_meetings = @standup_meeting_group.standup_meetings
                                               .includes(:user)
                                               .where(meeting_date: @meeting_date)
+    @current_user_standup_meeting = @standup_meetings.find do |meeting|
```

Comment:
small nit, can you use `detect`. Unsure if this is defined in our rubocop rules but `find`/`where` etc I associate with ActiveRecord while `detect` and `select` are arrays.